stable/vsphere-csi-driver/Chart.yaml

```
@@ -4,7 +4,7 @@ description: vSphere Container Storage Interface (CSI) driver
 name: vsphere-csi-driver
 maintainers:
   - name: sebbrandt87
-version: 1.0.0
+version: 1.0.1
```

This change justifies a minor version bump IMHO as it's adding functionality (in a backwards-compatible manner).
```suggestion
version: 1.1.0
```

---

🙆‍♂️